morphia/src/main/java/org/mongodb/morphia/query/Query.java

```diff
@@ -15,166 +15,142 @@
  * @author Scott Hernandez
```

Comment:
Not to beat a dead horse, but why are you doing this?  It's going to make diff-ing this file very painful in the future.